src/web/index.js

```diff
@@ -40,6 +41,7 @@ const jsonCodec = JsonUrl("lzma");
 
 const urlEncodeTheme = theme =>
   jsonCodec.compress(theme).then(value => {
+    console.log(theme); // eslint-disable-line
```

Comment:
Use the `log()` util function for stuff like this - it doesn't need eslint-disable-line and the output gets turned off in production builds